templates/allauth/account/login.html

```diff
@@ -15,27 +15,24 @@
 {% block allauth %}
 
     <h1>{% trans "Sign In" %}</h1>
-
     {% get_providers as socialaccount_providers %}
 
     {% if socialaccount_providers %}
-        <p>{% blocktrans with site.name as site_name %}Please sign in with one
-            of your existing third party accounts. Or, <a href="{{ signup_url }}">sign up</a>
-            for a {{ site_name }} account and sign in below:{% endblocktrans %}
+        <p>{% blocktrans with site.name as site_name %}
+                Please sign in with your Code Institute Slack!<br />
```

Comment:
`<br>` 😉 not `<br />`

Reply:
Changed this. It will take me some time until I will do this automatically :rofl: